src/SingleCellAnnotator.imjoy.html

```diff
@@ -755,14 +802,14 @@
           <button :disabled="current_image_index===0" class="btn transparent-btn" @click="previousImage()"><i class="icon icon-back"></i>&nbsp;Back</button>
           <button :disabled="current_image_data.discarded||annotation_mode" class="btn transparent-btn" @click="selectLocation(true)" :class="{'selected-location-btn': current_location_selected===true}"><i class="icon icon-check"></i>&nbsp;Yes</button>
           <button :disabled="current_image_data.discarded||annotation_mode" class="btn transparent-btn" @click="selectLocation(false)" :class="{'selected-location-btn': current_location_selected===false}"><i class="icon icon-cross"></i>&nbsp;No</button>
-          <button :disabled="current_image_index>=cached_image_data.length||((current_location_selected==null)&&current_image_data.annotation_status!=true)" class="btn transparent-btn" @click="nextImage()">Next&nbsp;<i class="icon icon-forward"></i></button>
+          <button :disabled="current_image_index>=cached_image_data.length||((current_location_selected===null)&&(current_image_data.annotation_status!==true))" class="btn transparent-btn" @click="nextImage()">Next&nbsp;<i class="icon icon-forward"></i></button>
         </div>
         <span class="h3 floating-label">{{current_image_data.location}}</span>
       </template>
       <div class="input-group floating-switch">
         <button class="btn" :class="{'transparent-btn': current_image_data.discarded!==true}" @click="toggleDiscard()">Discard</button>&nbsp;&nbsp;
         <label class="form-switch">
-          <input type="checkbox" v-model="annotation_mode" :disabled="current_image_data.mask_adjusted==false" @change="annotation_mode_changed">
+          <input type="checkbox" v-model="annotation_mode" :disabled="current_image_data.mask_adjusted===false" @change="annotation_mode_changed">
```

Comment:
```suggestion
          <input type="checkbox" v-model="annotation_mode" :disabled="!current_image_data.mask_adjusted" @change="annotation_mode_changed">
```

Reply:
not correct. have null value

Reply:
Well, you should only have `true` or (null || false), I don't see why you need to treat false and null differently.

Reply:
null and false is implemented in the script for key judgement. will change that in another PR. to avoid failure of the code

Reply:
I think you don't need that, it's not like python when the key doesn't exist, it throw error. In JS, you if there is no key, you can still use it, but the default value will be `undefined` (not `null`).